clients/src/test/java/org/apache/kafka/common/logging/Log4jControllerTest.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.kafka.common.logging;
+
+import org.apache.log4j.Level;
+import org.apache.log4j.Logger;
+import org.junit.Test;
+
+import java.util.Hashtable;
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+public class Log4jControllerTest {
+
+    @Test
+    public void testGetLoggers() {
+        Hashtable<String, Logger> x = new Hashtable<>();
+        Logger h1 = new Logger("h1"){};
+        h1.setLevel(Level.INFO);
+        Logger h2 = new Logger("h2"){};
+        h2.setLevel(Level.WARN);
+        x.put("h2", h1);
+        x.put("h1", h2);
+        Log4jController controller = mock(Log4jController.class);
+        when(controller.getLoggers()).thenCallRealMethod();
+        when(controller.getCurrentLevel(any())).thenCallRealMethod();
+        when(controller.currentLoggers()).thenReturn(x.elements());
+        when(controller.rootLogLevel()).thenReturn(Level.ERROR);
+
+        List<String> loggers = controller.getLoggers();
+
+        assertEquals("Expecting 3 loggers", 3, loggers.size());
+        assertEquals("Root loggers should be set at error level", "root=ERROR", loggers.get(0));
+        assertEquals("h1 should be ordered before h2", "h1=INFO", loggers.get(1));
+        assertEquals("h2 should be set at warn", "h2=WARN", loggers.get(2));
+    }
+
+    @Test
+    public void testGetLogLevel() {
+        Log4jController controller = mock(Log4jController.class);
+        when(controller.getLogLevel(any())).thenCallRealMethod();
+        Logger h = new Logger("hello"){};
+        when(controller.loggerByName("hello")).thenReturn(h);
+        when(controller.getCurrentLevel(any())).thenReturn(Level.WARN);
+
+        assertEquals("logger should be at WARN level", controller.getLogLevel("hello"), "WARN");
```

Comment:
nit: the signature for assertEquals is `assertEquals(Object expected, Object actual)` -- we want the expected value to be first, in this case "WARN"

Ditto for the test below